nexus-repository-conan/src/main/java/org/sonatype/repository/conan/internal/hosted/v1/HostedHandlers.java

```diff
@@ -112,14 +97,24 @@ private Response upload(final Context context, final String filename) throws IOE
   public final Handler packageSnapshot = context -> {
     State state = context.getAttributes().require(State.class);
     ConanCoords coord = convertFromState(state);
-    String path = getAssetPath(coord);
+    String path = getHostedAssetPath(coord);
 
     return context.getRepository()
             .facet(ConanHostedFacet.class)
             .getPackageSnapshot(path, context);
   };
 
-  private String getAssetPath(final ConanCoords coord) {
-    return V1_CONANS + getPath(coord);
+  // TODO will be removed
+  public static String getHostedAssetPath(final ConanCoords coord) {
+    return getHostedAssetPath(coord, null);
+  }
+
+  // TODO assetKind will be required
```

Comment:
Add ticket number or any additional info not to forget about this `TODO`

Reply:
fixed https://github.com/sonatype-nexus-community/nexus-repository-conan/pull/91/commits/c39317f5cef05bd0906093aed821cbcb993407a9